Roadmap.md

```diff
@@ -0,0 +1,81 @@
+
+Note: We should write tests as we fix these to ensure no regressions!
+
+👾 Small features (low hanging fruit)
+
+- remove [hard-coded gas price](https://github.com/aragon/aragon-cli/issues/353) (would be nice to let the user decide how much)
+
+👾 [Claiming an aragonid](https://github.com/aragon/aragon-cli/issues/347) 
+
+👾 Document new features (dao token commands)
+
+👾 Restructure documentation on hack.aragon to include:
+
+- Quick start / Overview / Introduction
+- Guides around specific subjects (e.g.: Publish to different environments, Using a different Ethereum account, Propagating content, etc), similar to [this](https://github.com/aragon/aragon-react-boilerplate#publish). Maybe also link more in-depth guides such as [this](https://forum.aragon.org/t/guide-custom-aragon-organization-deployment-using-the-cli/507)
+- FAQ section. We have [this](https://hack.aragon.org/docs/cli-usage.html#troubleshooting-faq) currently, but it should be it's own section and much more elaborate (If there is a recurring question on #dev-help we will add it there)
+
+## Long-term
+
+- Types (use Flow or Typescript) because it will allow us to catch more "errors" at compile time, having commands silently fail without noticing (like [this](https://github.com/aragon/aragon-cli/pull/334#discussion_r248659171)) less often. Using Typescript is also great for intellisense support (having auto-completion from IDEs) and refactoring
+- Improve smart contract tooling using new Ethereum dev tools from 0x:
+    - `[sol-trace](https://sol-trace.com)`: Human-readable stack traces
+    - `[sol-coverage](https://sol-coverage.com)`*:* Solidity code coverage
+    - `[sol-profiler](https://sol-profiler.com)`: Gas profiling for Solidity
+    - `[sol-compiler](https://sol-compiler.com)`: Solidity compilation
+
+    We plan to prioritize `sol-trace` and to look into the benefits that the others tools could have on the developer experience and decide whether to work on that *short-term*.
+
+- Identify contracts like the `MiniMeTokenFactory` which do not need to be deployed every time, but rather reused, and publish them to `aragonpm.eth` (just like the `dao-kits`)
+
+- Wizard onboarding 🧙‍♂️
+    - The goal of this tool will be to help new users create a custom DAO
+    - "Import" the onboarding of the multisig and democracy kits from the ui client
+    - Improve UX for installing "core" apps (e.g.: TokenManager, Voting, etc.), because currently is a bit tricky as you have to check what arguments the `initialize` function takes from the app's contract and pass them with `--app-init-args`, e.g.: `dao install [dao-address] voting --app-init-args 0x00000001 600000000000000000 250000000000000000 604800`. We should do this, instead of the users, so they can see what are the needed parameters with `--help`  or `--interactive`. For example `dao install voting --help` could return `dao install voting [dao-address] [token-address] [support-required] [quorum] [vote-time]`
```

Comment:
**Brett:** We should be able to do get the initialize function's params from the artifact.json if it's published onto an aragonPM instance